sql/pgtap.sql.in

```diff
@@ -11366,3 +11390,65 @@ RETURNS TEXT AS $$
         'Function ' || quote_ident($1) || '() should not be a procedure'
     );
 $$ LANGUAGE sql;
+
+-- table_comment_has(schema, table, comment, description)
+CREATE OR REPLACE FUNCTION table_comment_has(TEXT, TEXT, TEXT, TEXT)
```

Comment:
WE use `_has` more to indicate that a database has an object, like a table. I think a better term to use here would be `_contains`. Or perhaps `_matches` and use a regular expression? That could help avoid variations in line endings across platforms.